### ISSUE TITLE
Strange veins

### ISSUE THREAD

**Issue by XieJiajian2024** [icon-lab/ResViT#26]
Thank you for your great work! I used my own data set to train your model. During the training process, the pictures of checkpoints/name/web/images are normal full black background, but when testing, use
 visuals = model.get_current_visuals()  # get image results
 img_path = model.get_image_paths()  # get image paths
visualizer.save_images(webpage, visuals, img_path)
There are many strange lines on the background of the saved test chart. What is the possible reason? Thanks again

<img width="512" height="512" alt="Image" src="https://github.com/user-attachments/assets/a66371e3-938e-4772-9e05-99d25e76f167" />

**Comment by XieJiajian2024** [comment]
this is input:

<img width="512" height="512" alt="Image" src="https://github.com/user-attachments/assets/f1a1b653-dd34-4c0a-9a23-e1dad0e43f27" />

**Comment by XieJiajian2024** [comment]
When discarding the officially defined from util.visualizer import visualizer and using the following custom save image code:
def normalize_image(img):
    with torch.no_grad():
        print(img.min(), img.max())
        img = (img + 1.0) * 0.5#model output[-1,1]
        img = torch.clamp(img, 0.0, 1.0)
        return img
def save_triplet(real_A, fake_B, real_B, img_path, save_root, opt):
    src_file = img_path[0]
    basename = os.path.splitext(os.path.basename(src_file))[0]
    sub_dir  = os.path.dirname(src_file).replace(opt.dataroot, '').lstrip(os.sep)
    out_dir  = os.path.join(save_root, sub_dir)
    os.makedirs(out_dir, exist_ok=True)
    def tensor2pil(ti, name):
        ti = ti[0, 0].cpu()
        if ti.min() < 0 or ti.max() > 1:
            ti=normalize_image(ti)
        img_np = (ti * 255).round().numpy().astype(np.uint8)
        return Image.fromarray(img_np, mode='L')
        return ti

    target_size = None
    if opt.loadSize != 0:
        target_size = (opt.loadSize, opt.loadSize)

    for tag, tensor in [('realA', real_A), ('fakeB', fake_B), ('realB', real_B)]:
        pil = tensor2pil(tensor, tag)
        if target_size and pil.size != target_size:
            pil = pil.resize(target_size, Image.BILINEAR)
        save_name = os.path.join(out_dir, f'{basename}_{tag}.png')
        pil.save(save_name, compress_level=0)

strange stripes still happen. Do you have any suggestions.thanks you
